rasa/core/policies/ted_policy.py

```diff
@@ -1173,6 +1217,18 @@ def __init__(
             entity_tag_specs: the entity tag specifications
         """
         super().__init__("TED", config, data_signature, label_data)
+        try:
+            # Disable all GPUS
+            tf.config.set_visible_devices([], 'GPU')
+            visible_devices = tf.config.get_visible_devices()
+            for device in visible_devices:
+                assert device.device_type != 'GPU'
+        except:
+            # Invalid device or cannot modify virtual devices once initialized.
+            pass
+        import os
+        os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
```

Comment:
I think this might disable GPU use for everything that follows TED training - within the same python session - which would break cases where we run inference right after training (and expect GPU to be used for that).

Reply:
Yeah, this is an old version i have tested. Sorry, I forgot to exclude.